x-pack/plugins/graph/tsconfig.json

```diff
@@ -25,6 +25,7 @@
     { "path": "../../../src/plugins/home/tsconfig.json"},
     { "path": "../../../src/plugins/kibana_utils/tsconfig.json" },
     { "path": "../../../src/plugins/kibana_react/tsconfig.json" },
+    { "path": "../../../src/plugins/presentation_util/tsconfig.json" },
```

Comment:
@kertal Is this a left over, or really needed for anything? (Since it's in the Graph plugins tsconfig